docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -0,0 +1,257 @@
+
+## Create the add-in
+
+You can create an Office Add-in with a JSON manifest by using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md). The Yeoman generator creates a Node.js project that can be managed with Visual Studio Code or any other editor.
+
+### Prerequisites
+
+[!include[Set up requirements](../includes/set-up-dev-environment-beforehand.md)]
+
+- [.NET runtime](https://dotnet.microsoft.com/download/dotnet/6.0/runtime) for Windows. One of the tools used in the preview runs on .NET.
+
+[!INCLUDE [Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
+
+- [Visual Studio Code (VS Code)](https://code.visualstudio.com/) or your preferred code editor
+
+- Outlook on Windows (connected to a Microsoft 365 account)
+
+### Create the add-in project
+
+1. [!include[Yeoman generator create project guidance](../includes/yo-office-command-guidance.md)]
+
+    - **Choose a project type** - `Outlook Add-in with Teams Manifest (Developer preview)`
+
+    - **What do you want to name your add-in?** - `Outlook Add-in with Teams Manifest`
+
+     ![Screenshot showing the prompts and answers for the Yeoman generator in a command line interface with JSON manifest option chosen.](../images/yo-office-outlook-json-manifest.png)
+
+    After you complete the wizard, the generator will create the project and install supporting Node components.
+
+    [!include[Yeoman generator next steps](../includes/yo-office-next-steps.md)]
+
+1. Navigate to the root folder of the web application project.
+
+    ```command&nbsp;line
+    cd "Outlook Add-in with Teams Manifest"
+    ```
+
+### Explore the project
+
+The add-in project that you've created with the Yeoman generator contains sample code for a very basic task pane add-in.
+
+- The **./manifest.json** file in the root directory of the project defines the settings and capabilities of the add-in.
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane.
+- The **./src/taskpane/taskpane.css** file contains the CSS that's applied to content in the task pane.
+- The **./src/taskpane/taskpane.ts** file contains code that calls the Office JavaScript library to facilitate interaction between the task pane and Outlook.
+- The **./src/command/command.html** file will be edited by WebPack at build time to insert an HTML `<script>` tag that loads the JavaScript file that is transpiled from the command.ts file.
+- The **./src/command/command.ts** file has little code in it at first. Later in this article, you'll add code to it that calls the Office JavaScript library and that executes when a custom ribbon button is selected.
+
+### Update the code
+
+1. Open your project in VS Code or your preferred code editor.
+   [!INCLUDE [Instructions for opening add-in project in VS Code via command line](../includes/vs-code-open-project-via-command-line.md)]
```

Comment:
This reference cites how to work with Mac and mac is not supported, so I think this could lead to confusion